checks/runchecks.go

```diff
@@ -11,15 +13,60 @@ import (
 	"github.com/flanksource/commons/logger"
 )
 
-func RunChecks(ctx *context.Context) []*pkg.CheckResult {
+// A list of check types that are permanently disabled.
+var disabledChecks map[string]struct{}
+
+func getDisabledChecks(ctx *context.Context) (map[string]struct{}, error) {
+	if disabledChecks != nil {
```

Comment:
I might make sense to use a  5 minute cache here

Reply:
@moshloop don't we just need to read this once (on startup)?

Reply:
The startup is more for licensing / bootstraping,  If one particular check type is causing issues in production, then we might want to disable it via update to properties